apps/forklift/lib/forklift/event/event_handler.ex

```diff
@@ -145,6 +169,20 @@ defmodule Forklift.Event.EventHandler do
       :discard
   end
 
+  def handle_event(%Brook.Event{type: ingestion_delete(), data: %SmartCity.Ingestion{} = ingestion, author: author}) do
+    Logger.info("Dataset #{ingestion.id} - Received ingestion_delete event from #{author}")
+
+    ingestion_delete()
+    |> add_event_count(author, ingestion.id)
+
+    Forklift.Ingestions.delete(ingestion.id)
+  rescue
+    error ->
+      Logger.error("ingestion_delete failed to process.")
+      DeadLetter.process([ingestion.id], nil, ingestion, Atom.to_string(@instance_name), reason: inspect(error))
```

Comment:
First param should be the ingestion's target datasets. 2nd param should be the ingestion ID